src/zenml/artifacts/artifact_config.py

```diff
@@ -47,8 +48,9 @@ def my_step() -> Annotated[
         version: The version of the artifact.
         tags: The tags of the artifact.
         run_metadata: Metadata to add to the artifact.
-        is_model_artifact: Whether the artifact is a model artifact.
-        is_deployment_artifact: Whether the artifact is a deployment artifact.
+        artifact_type: Optional type of the artifact. If not given, the type
+            specified by the materializer that is used to save this artifact
+            is used.
```

Comment:
Maybe show how to use this in a docstring above?

Reply:
Added it